substrate/frame/election-provider-multi-block/src/unsigned/miner.rs

```diff
@@ -221,14 +221,23 @@ pub type PageSupportsOfMiner<T> = frame_election_provider_support::BoundedSuppor
 	<T as MinerConfig>::MaxBackersPerWinner,
 >;
 
+/// Helper type that computes the maximum total winners across all pages.
+pub struct MaxWinnersTotal<T: MinerConfig>(core::marker::PhantomData<T>);
```

Comment:
Elsewhere in the code, we have a similar naming dilemma: 

- `MaxBackersPerWinner`: page-local bound
- `MaxBackersPerWinnerFinal`: bound across all pages

I am happy with either naming schemes (or even a new one) but I do suggest them to be consistent.